src/components/FormFields/SequenceOrderField/SequenceTransferTitle.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const headerStyle = {
```

Comment:
is there a reason to put the style here in the component rather than in a css module?

Reply:
Fixed in a3c8b45